Gemfile

```diff
@@ -25,7 +25,7 @@ gem 'bcrypt', '~> 3.1.7'
 
 gem 'slim-rails'
 gem 'webpacker'
-
+gem 'tod'
```

Comment:
:memo: [jackc/tod: Time of day and shift types for Ruby](https://github.com/jackc/tod)